config/database.yml

```diff
@@ -1,20 +1,25 @@
-# SQLite version 3.x
-#   gem install sqlite3
-#
 #   Ensure the SQLite 3 gem is defined in your Gemfile
 #   gem 'sqlite3'
-development:
-  adapter: postgresql
-  encoding: utf-8
-  database: collated_areas_development
+#
+default: &default
+  adapter: postgis
   pool: 5
-  username: postgres
-  password:
+  timeout: 5000
+  host: <%= ENV['POSTGRES_HOST'] %>
+  username: <%= ENV['POSTGRES_USER'] %>
+  password: <%= ENV['POSTGRES_PASSWORD'] %>
+  template: template_postgis
```

Comment:
Not sure what this template is for.
Also, I wonder why you have got those as ENV variables.

Looking at the PP database config file, that is a bit simpler then this. I would keep that structure for consistency.

https://github.com/unepwcmc/ProtectedPlanet/blob/master/config/database.yml

Reply:
Sorry @Levia I accidentally committed my Docker specific database settings here.

Reply:
@Levia would it be acceptable to simply copy and paste the database yml file from PP as I have the problem that I cannot run PP natively so I need that Docker ENV var style configuration to get it working locally. At least I know that the PP database yml is working.

Reply:
@drewpotter Yeah I guess that's fine. The db configuration should be the same anyway